OpenRA.Game/Traits/World/ResourceType.cs

```diff
@@ -64,6 +69,18 @@ public class ResourceTypeInfo : ITraitInfo
 		[Desc("Harvester content pip color.")]
 		public PipType PipColor = PipType.Yellow;
 
+		[Desc("Resource growth rate (seconds per growth)")]
+		public int GrowthRate = 37;
+
+		[Desc("Resource growth percent (% of max density)")]
+		public float GrowthPercent = 0.09f;
+
+		[Desc("Resource spread rate (seconds per spread)")]
+		public int SpreadRate = 37;
+
+		[Desc("Resource spread percent (% of max density)")]
+		public float SpreadPercent = 0.09f;
```

Comment:
We eliminated floats from simulation code several months ago because they were amongst the main desync sources. I'd prefer an integer percentage here, pinging @pchote for 2nd opinion.